frontend/src/utils/useDefaultDate.ts

```diff
@@ -25,10 +28,23 @@ export function useDefaultDate(layerId: LayerKey): number | undefined {
   const { updateHistory } = useUrlHistory();
 
   // TODO - use getPossibleDatesForLayer
+  // Update on 2024-07-01: getPossibleDatesForLayer would give us the first possible layer date, we want the _last_ possible date.
```

Comment:
Note here, I don't think this to do is still relevant. Either that or we should refactor how composite layers are handled in getPossibleDatesForLayer